test/deploy-pipeline-lite.sh

```diff
@@ -41,7 +41,7 @@ if [ -z "$KFP_DEPLOY_RELEASE" ]; then
   echo "Deploying KFP in working directory..."
   KFP_MANIFEST_DIR=${DIR}/manifests
 
-  pushd ${KFP_MANIFEST_DIR}/crd
```

Comment:
@Bobgy The reason CRD is split from the rest is due to eventual consistency deploying  CRD. 
This was encountered when we introduced the KFP application CRD.
Now the split criterial becomes whether a resource is cluster-level or not. this is fine to me but just want to call out that this was not the initial intent.

Reply:
Ack. Thanks for the call out.
That's true, now we are separating cluster resources for two irrelevant reasons.